_data/simple-icons.json

```diff
@@ -8827,8 +8827,8 @@
         },
         {
             "title": "Lerna",
-            "hex": "334155",
-            "source": "https://github.com/lerna/website/blob/dcef9c248b2b67bdd2214058beca437732690226/images/lerna-logo-dark.svg"
+            "hex": "9333EA",
+            "source": "https://user-images.githubusercontent.com/900523/173044458-fd0b57f6-6374-4265-98b5-eb8f55fe1fb3.svg"
```

Comment:
I don't think we should use user uploaded images as sources like this. Where is this taken from?

Reply:
From their GitHub readme, uploaded by their owner. There are no more links for it. It's the most accurate one.

GitHub asset links are always permanent. So it's OK to use the `user-images.githubusercontent` link.